test/src/test/kotlin/graphql/nadel/tests/CentralSchemaTesting.kt

```diff
@@ -0,0 +1,199 @@
+package graphql.nadel.tests
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import graphql.language.AstPrinter
+import graphql.nadel.Nadel
+import graphql.nadel.NadelExecutionInput.newNadelExecutionInput
+import graphql.nadel.NextgenEngine
+import graphql.nadel.ServiceExecution
+import graphql.nadel.ServiceExecutionFactory
+import graphql.nadel.ServiceExecutionResult
+import graphql.nadel.enginekt.util.JsonMap
+import graphql.nadel.schema.NeverWiringFactory
+import graphql.schema.idl.SchemaParser
+import graphql.schema.idl.TypeDefinitionRegistry
+import kotlinx.coroutines.future.asDeferred
+import java.io.File
+import java.util.concurrent.CompletableFuture
+import kotlin.system.measureTimeMillis
+
+/**
+ * Run this and replace the path to central schema to load it up in Nadel.
+ */
+suspend fun main() {
+    val schema = File(
+        "/Users/fwang/Documents/GraphQL/graphql-central-schema/schema/",
```

Comment:
If you want to feed central schema straight into Nadel, use this class and change this path to your repo.